src/linter.cc

```diff
@@ -40,6 +40,87 @@
 // Implemented rules in the same order with rules in the documention.
 namespace zetasql::linter {
 
+absl::string_view GetNodeString(const ASTNode *node,
+                                const absl::string_view &sql) {
+  const int &start = node->GetParseLocationRange().start().GetByteOffset();
+  const int &end = node->GetParseLocationRange().end().GetByteOffset();
+  return sql.substr(start, end - start);
+}
+
+bool IsUpperCamelCase(absl::string_view name) {
+  if (name.size() > 0 && !('A' <= name[0] && name[0] <= 'Z')) return false;
```

Comment:
Please add a helper function like the following and use them here to improve readability. 
bool IsUppercase(char c) { return c >= 'A' && c <= 'Z'; }

bool IsLowercase(char c) { return c >= 'a' && c <= 'z';

Reply:
Done.